etc/firewall-firewalld/syncthing.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Syncthing</short>
+  <description>Syncthing file synchronization</description>
+  <port protocol="udp" port="22000"/>
```

Comment:
You also need to allow tcp/22000 and locally, 21027/udp (if there is such a thing, in firewalld) as per the [firewall docs](https://docs.syncthing.net/users/firewall.html#firewall-setup).